src/NzbDrone.Core/Notifications/Discord/Discord.cs

```diff
@@ -317,6 +317,29 @@ public override void OnHealthIssue(HealthCheck.HealthCheck healthCheck)
             _proxy.SendPayload(payload, Settings);
         }
 
+        public override void OnHealthRestored(HealthCheck.HealthCheck previousCheck)
+        {
+            var attachments = new List<Embed>
+                              {
+                                  new Embed
+                                  {
+                                      Author = new DiscordAuthor
+                                      {
+                                          Name = Settings.Author.IsNullOrWhiteSpace() ? Environment.MachineName : Settings.Author,
+                                          IconUrl = "https://raw.githubusercontent.com/Sonarr/Sonarr/develop/Logo/256.png"
+                                      },
+                                      Title = "Successful " + previousCheck.Source.Name,
```

Comment:
`Health Issue Resolved: previousCheck.Source.Name` to match other cases? `Successful` doesn't really cover it I feel.